jcl/src/java.base/share/classes/openj9/internal/criu/InternalCRIUSupport.java

```diff
@@ -96,6 +97,15 @@ public static long getLastRestoreTime() {
 		return getLastRestoreTimeImpl();
 	}
 
+	/**
+	 * Get the start time of the CRIU restore process.
+	 *
+	 * @return the time in nanoseconds since the epoch, -1 if restore has not occurred.
+	 */
+	public static long getCRIURestoreStartTime() {
```

Comment:
i think renaming this to `getCRIUProcessRestoreStartTime` would be more clear

Reply:
I think the documentation could be improved so the distinction between `getCRIUProcessRestoreStartTime()` and `getLastRestoreTime()` is clearer.

Both methods or neither method should have "CRIU" included in name. My preference is to omit "CRIU": it is implied by the class name.

What is the point of those two methods anyway? I only see uses in tests.

Reply:
I renamed:
- `getCRIURestoreStartTime` to `getCRIUProcessRestoreStartTime`
- `getCRIURestoreStartTimeImpl` to `getCRIUProcessRestoreStartTimeImpl`
- `criuRestoreStartTimeInNanoseconds` to `criuProcessRestoreStartTimeInNanoseconds`
- `testGetCRIURestoreStartTime` to `testGetCRIUProcessRestoreStartTime`
- `criuRestoreStartTime` to `criuProcessRestoreStartTime`

Reply:
I improved the documentation for `getCRIUProcessRestoreStartTime()` to emphasize its distinction from `getLastRestoreTime()`. My rationale for keeping `CRIU` in the method name was to indicate that the start time is associated with the criu process rather than the restored java process. It seems like the method name `getProcessRestoreStartTime()` could be ambiguous.

The purpose of `getCRIUProcessRestoreStartTime()` and `getLastRestoreTime()` is to know the latency of the criu process before it restores the java process, to improve debugging, and to help with the work of supporting openjdk crac in openj9, potentially aiding in performance comparisons between criu and openjdk crac.

Reply:
First, I do think it is an improvement that all methods that return a time (delta) use units of nanoseconds.

I repeat my preference of avoiding redundancy in names in the API of a class: in my opinion there's no need to repeat CRIU in `InternalCRIUSupport.getCRIUProcessRestoreStartTime()` (you'll notice CRIU only appears once in most other uses, e.g. `InternalCRIUSupport.checkpointJVM()`, `InternalCRIUSupport.registerPreCheckpointHook()`, etc.

No one has pointed out uses of `getCRIUProcessRestoreStartTime()` or `getLastRestoreTime()` outside of test code. If there will be none, then I maintain that this is not the right place to provide access to that information (it can go in native code specific to the relevant tests).

Reply:
> No one has pointed out uses of getCRIUProcessRestoreStartTime()

The use cases havent been implemented yet but they do exist. A couple examples are replacing Liberties script to measure total restore time (CRIU + JVM). Second is supporting CRACMXBeans which require these metrics. The prupose of this PR is to get the fundamental pieces ready for improvements elsewhere.